lib/web/apiserver.go

```diff
@@ -2127,22 +2149,12 @@ func (h *Handler) siteSessionGet(w http.ResponseWriter, r *http.Request, p httpr
 		return nil, trace.Wrap(err)
 	}
 
-	sess, err := clt.GetSession(apidefaults.Namespace, *sessionID)
+	tracker, err := clt.GetSessionTracker(r.Context(), string(*sessionID))
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
 
-	// DELETE IN: 5.0.0
-	// Teleport Nodes < v4.3 does not set ClusterName, ServerHostname with new sessions,
-	// which 4.3 UI client relies on to create URL's and display node inform.
-	if sess.ClusterName == "" {
-		sess.ClusterName = p.ByName("site")
-	}
-	if sess.ServerHostname == "" {
-		sess.ServerHostname = sess.ServerID
-	}
-
-	return *sess, nil
+	return trackerToLegacySession(tracker, site.GetName()), nil
```

Comment:
Why in `siteSessionsGet` sessions that are not  `types.SSHSessionKind`  are filter out but `siteSessionGet` function return any kind of the session ?

Reply:
Added a check for this now.